2d_motion/main_2dconversion.m

```diff
@@ -2,41 +2,42 @@
 % This is a script that makes the 2d motion, it'll become a function at
 % some point
 clear;close all;clc;
-%% Generate Orbits
+addpath('C:\Users\Jason\Documents\ASEN4018\git\cast\main')
```

Comment:
addpath('../main') would be appropriate, better would be to move the main script into the main folder and just call the importCast script.